sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotOverwriteDictionaryValues.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Tests.Diagnostics
+{
+    class TestCases
+    {
+        void Indexers(Dictionary<int, int> items)
+        {
+            int index;
+
+            items[index] = 1; // Secondary
```

Comment:
Is there a way to check that the secondary location is correct?

Reply:
If you are asking about the underline syntax, yes, we can.